__main__.py

```diff
@@ -74,7 +74,7 @@
 
 @bot.command()
 async def scheddy(ctx, league_name):
-    await command.scheddy.handle(ctx, league_name, bb_db)
+    await command.scheddy.handle(ctx, bb_db, league_name)
```

Comment:
so while this allows the application to run now following the addition of a default parameter to `command.scheddy.handle` the default also needs to be added to the `scheddy` command signature here, otherwise the command submitted to the bot still requires it, and ommision results in an exception from the discordpy api.
I'll sort this out today but thought it best to leave a comment explaining here, cos this change isnt really a fix - like I said last night, I became aware but didnt fix it at the time due to not expecting collaboration =)
i'm actually considering ommiting the default, or at least making it a configureable in the settings.